src/environments/environment.ts

```diff
@@ -5,6 +5,6 @@
 export const environment = {
   production: false,
   profileUrl:
-    'https://landville-backend-web-api.herokuapp.com/api/v1/auth/profile/',
-  api_url: 'https://landville-backend-web-api.herokuapp.com/api/v1',
+    'https://landville-backend-web-api.herokuapp.com/api/v1/auth/auth/profile/',
```

Comment:
hey @Genza999, I see this has changed, we should verify that the profile related features are still working and that this is the correct backend profile link.

Reply:
Thanks @CryceTruly , I have worked on this!